### PR TITLE
Show message pinned at the bottom of the window warning of cookie usage to comply with EU law

### PR DESCRIPTION
**Pull request by Kiloutre**
No preview because i haven't designed anything yet, will do once it gets merged

### CODE REVIEW

File: templates/template_test.go [hunk 1]
```diff
@@ -288,6 +288,7 @@ func mockupCommonvariables(t *testing.T) jet.VarMap {
 	variables.Set("User", &models.User{})
 	variables.Set("URL", &url.URL{})
 	variables.Set("CsrfToken", "xxxxxx")
+	variables.Set("EUCookieLaw", "test")
```
Comment (akuma06) [inline templates/template_test.go]: Set this to a boolean

File: utils/publicSettings/publicSettings.go [hunk 2]
```diff
@@ -237,6 +239,18 @@ func GetMascotURLFromRequest(c *gin.Context) string {
 	return ""
 }
 
+func GetEUCookieFromRequest(c *gin.Context) bool {
+	cookie, err := c.Cookie("EU_Cookie")
```
Comment (akuma06) [inline utils/publicSettings/publicSettings.go]: Since you don't use cookie, you just have to do:
_, err := c.Cookie("EU_Cookie")

**Review by akuma06 — CHANGES_REQUESTED** [review]

File: templates/template_test.go [hunk 3]
```diff
@@ -288,7 +288,7 @@ func mockupCommonvariables(t *testing.T) jet.VarMap {
 	variables.Set("User", &models.User{})
 	variables.Set("URL", &url.URL{})
 	variables.Set("CsrfToken", "xxxxxx")
-	variables.Set("EUCookieLaw", "test")
+	variables.Set("EUCookieLaw", "false")
```
Comment (akuma06) [inline templates/template_test.go]: No I mean, don't use "false", but `false`

Comment (Kiloutre) [reply templates/template_test.go]: i know it was just a mistake

**Review by akuma06 — APPROVED** [review]

**Comment by yiiTT** [comment]
Kilo you really like to remove coverage

**Pull request merged**